pvacseq/lib/main.py

```diff
@@ -77,40 +89,42 @@ def main(args_input = sys.argv[1:]):
     )
     print("Completed")
 
-    netmhc_output = set(
-        run([args.netmhc_path+" -A"],
-            shell=True, stdout=PIPE
-            ).stdout.decode().split('\n')
-        )
+#    netmhc_output = set(
+#        run([args.netmhc_path+" -A"],
+#            shell=True, stdout=PIPE
+#            ).stdout.decode().split('\n')
+#        )
 
-    for epl in args.epitope_length:
-        for a in args.allele:
-            if a in netmhc_output:
-                net_out = ".".join([args.sample_name, a, str(epl), "netmhc.txt"])
-                print("Running NetMHC on Allele", a,
-                      "and Epitope Length", epl
-                      )
-                netmhc_cmd = "%s -a %s -l %d %s -x %s" % (
-                    args.netmhc_path, a, epl,
+    iedb_output_files = []
+    for method in args.prediction_algorithms:
+        iedb_method = prediction_method_lookup(method)
+        for epl in args.epitope_length:
+            for a in args.allele:
+ #               if a in netmhc_output:
+                iedb_out = os.path.join(args.output_dir, ".".join([args.sample_name, a, str(epl), iedb_method, "tsv"]))
+                print("Running IEDB on Allele %s and Epitope Length %s with Method %s" % (a, epl, method))
+                lib.call_iedb.main([
```

Comment:
call_iedb doesn't write do its output file until after the HTTP request succeeds so that shouldn't be a problem.